src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -16,9 +16,8 @@
  * Provide default OG selection handler.
  *
  * @EntityReferenceSelection(
- *   id = "default:og",
+ *   id = "og:default",
```

Comment:
This change is :+1: from me :)
On 15 Nov 2015 14:48, "Roy Segall" <notifications@github.com> wrote:

> In src/Plugin/EntityReferenceSelection/OgSelection.php
> <https://github.com/amitaibu/og/pull/11#discussion_r44872639>:
>
> > @@ -16,9 +16,8 @@
> >   * Provide default OG selection handler.
> >   *
> >   * @EntityReferenceSelection(
> > - *   id = "default:og",
> > + *   id = "og:default",
>
> The idea is that we will take the original query object provided by the
> original selection handler(could be for node, term, user or even commerce
> product) and apply OG additional logic.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/amitaibu/og/pull/11/files#r44872639>.
>